src/FieldWarning/Assets/Terrain/Scripts/TerrainMap.cs

```diff
@@ -146,10 +137,22 @@ public TerrainMap(Terrain[] terrains1D)
         }
 
         _loader = new Loading("Terrain");
-        _loader.AddWorker(LoadHeightMap, "Loading height map");
+
+        //TODO create some debug UI to dump the map when needed
+        // TODO need to also somehow verify the height map is valid?? not sure how to do this each time without reading the original data.
```

Comment:
And a tiny optional fix, if you could break up lines at 80 or so (so the github split diff view doesn't have to word wrap automatically) I would appreciate it. Just so two files can be opened side by side and fit. In github, open 'files changed', go to options and press 'split' to see what I mean.

You also have a bunch of statements that aren't spaced on both sides, e.g. 'a+= b' (should be 'a += b')